pkg/xds/template/raw.go

```diff
@@ -0,0 +1,38 @@
+package template
```

Comment:
not sure about placement of this file. I could not put it into `envoy` package because of the cyclic import.

Reply:
Same comment as above, let's avoid dependency `pkg/core/resources/apis/mesh => pkg/xds/template`.

Maybe `pkg/util/envoy` ?